common/launch.go

```diff
@@ -102,11 +102,12 @@ func (l *LaunchOptions) Parse(ctx context.Context, opts goja.Value) error {
 				l.Headless = opts.Get(k).ToBoolean()
 			case "ignoreDefaultArgs":
 				v := opts.Get(k)
-				switch v.ExportType() {
-				case reflect.TypeOf(goja.Object{}):
-					args := v.Export().([]string)
-					l.IgnoreDefaultArgs = append(l.IgnoreDefaultArgs, args...)
+				var args []string
+				err := rt.ExportTo(v, &args)
+				if err != nil {
+					return fmt.Errorf("ignoreDefaultArgs should be an array of strings: %w", err)
```

Comment:
Do we want to support passing `true` as well, [to be consistent with Playwright](https://playwright.dev/docs/api/class-browsertype#browser-type-launch)?

The use case for that is dubious, and we don't need it to fix #536, but we might as well while we're here.

Reply:
I will open issue for that as I don't want to get this in this PR anymore and I feel like parsing it as a boolean as well will involve some code I don't particularly want to write :grimacing:

Reply:
https://github.com/grafana/xk6-browser/issues/549